.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -124,6 +126,10 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
 - [ ] If a new CSS style is added I verified that:
     - [ ] A similar style doesn’t already exist
     - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
+- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
+- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
```

Comment:
I think it is the doc like this, but looks inconsistent. Why these two are not nested in `If a new CSS style is added I verified that:`, and in the Author's checklist they are?

<img width="851" alt="image" src="https://user-images.githubusercontent.com/87341702/159505571-0f8d7807-5dc1-4446-8a57-691e191ae9c0.png">

Reply:
Oh good catch Aldo, thanks! I definitely have an issue with indentations 😅